assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -142,6 +149,8 @@ function MediaPane(props) {
 
   const [isPermissionDialogOpen, setIsPermissionDialogOpen] = useState(false);
 
+  const isSearching = Boolean(searchTerm.trim());
```

Comment:
`searchTerm` is always trimmed in the `onSearch` callback, so it shouldn't be needed here.

```suggestion
  const isSearching = searchTerm.length > 0;
```

Reply:
Good catch.